src/ecalc/libraries/libecalc/common/libecalc/core/graph_result.py

```diff
@@ -343,3 +388,43 @@ def get_results(self) -> EnergyCalculatorResult:
             emission_results=self.emission_results,
             variables_map=self.variables_map,
         )
+
+    def get_pressures_from_temporal_models(
+        self, energy_usage_model: dict, regularity: dict, pressure_type: CompressorPressureState
```

Comment:
not sure if `state` is a good name. perhaps `position` ? Ask a specialist of what it is called :)